code/ai/aigoals.cpp

```diff
@@ -2311,7 +2418,7 @@ void ai_process_mission_orders( int objnum, ai_info *aip )
 	object	*objp = &Objects[objnum];
 	object	*other_obj;
 	ai_goal	*current_goal;
-	int		wingnum, shipnum;
```

Comment:
What happened to `shipnum`?

Reply:
It's no longer needed (and in fact it was flagged by clang as an unused variable) because the current goal's ship is now accessed via the ship registry rather than an index into `Ships[]`.